stdlib/cmake/modules/AddSwiftStdlib.cmake

```diff
@@ -926,11 +926,14 @@ function(add_swift_target_library_single target name)
     endif()
   endif()
 
-  # FIXME: swiftDarwin currently trips an assertion in SymbolGraphGen
-  if (SWIFTLIB_IS_STDLIB AND SWIFT_STDLIB_BUILD_SYMBOL_GRAPHS AND NOT ${name} STREQUAL "swiftDarwin")
+  # FIXME: swiftDarwin and swiftDifferentiationUnittest currently trip an assertion in SymbolGraphGen
+  if (SWIFTLIB_IS_STDLIB AND SWIFT_STDLIB_BUILD_SYMBOL_GRAPHS AND NOT ${name} STREQUAL "swiftDarwin"
```

Comment:
Yikes. I'm okay with extending this existing hack, but do we know what's going on here?

Reply:
I haven't had the opportunity to look into this. I assume there's some kind of construction that we're trying to handle incorrectly, but i have dug into what and why.